myselfmarish/DynamicTeamPage
# Removing and adding images

## What does the PR do?
* It removes and adds images

## How do you check this PR?
* Pull down the branch
*  See images 

## Screenshots 
None.
